hazelcast/src/test/java/com/hazelcast/internal/nearcache/NearCacheRecordStoreTestSupport.java

```diff
@@ -144,23 +145,16 @@ protected void statsCalculated(InMemoryFormat inMemoryFormat) {
         }
 
         nearCacheRecordStore.clear();
-
-        switch (inMemoryFormat) {
-            case BINARY:
-            case OBJECT:
-                assertEquals(0, nearCacheStats.getOwnedEntryMemoryCost());
-                break;
-        }
+        assertEquals(0, nearCacheStats.getOwnedEntryMemoryCost());
```

Comment:
`HiDensityNearCacheRecordStoreTest` in enterprise calls this method with `InMemoryFormat.NATIVE`. I think this will break that test.